server/data/restClient.ts

```diff
@@ -74,9 +73,6 @@ export default class RestClient {
       return raw ? result : result.body
     } catch (error) {
       const sanitisedError = sanitiseError(error)
-      if (handle404 === true && error?.response?.status === 404) {
```

Comment:
Was this only used by that method? May be useful for other methods? I guess we can remove if it's not used.

Reply:
it was only used in prison-api class and so not applicable anymore since we need to "handle" 403 and 404 together. i think trapping 404s _could_ indeed be useful in general. maybe we can have a go at putting something like that into the template’s `RestClient` – but maybe give it a more descriptive name, like `returnNullOn404` or something